_posts/2020-11-23-boost-your-productivity-with-tailwind-css.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title: Boost your productivity with Tailwind CSS
+author: tuk
+excerpt: Tailwind CSS has proven its worth, and it's time you adopt it into your front-end toolkit. Let this post be your first step in doing so.
+tags:
+ - Software Development
+ - frontend
+ - CSS
+---
+
+If you've been following the front-end development scene lately, you may have heard a mention or two about [Tailwind CSS](https://tailwindcss.com/). The project has gained significant ground, and the trend seems to continue in its favor. In this blog post, I'll explain:
+
```

Comment:
Since this is the 'hook' part, where we want to entice people to read the blog, is it possible to get here some impact/benefit of reading this/using tailwind, in addition to saying it's trendy?

Doesn't need to be much, but if it has gained ground, and is trendy, does it make life easier/more fun/cleaner/faster? Why should I swap to it? Lengthy explanations can be later, but if we can drop a bold claim of the impact in the beginning, it might get a few more people to get interested and read it, even if they have not been following CSS scene lately.